civievent-widget.php

```diff
@@ -207,134 +207,45 @@ public function widget( $args, $instance ) {
 
 		if ( version_compare( $this->_civiversion, '4.3.alpha1' ) < 0 ) { return; }
 
-		$fields = $this->getFields();
+		$customDisplay = $this->validCustomDisplayFields( $instance );
+		$customDisplayFields = array_intersect_key( self::getCustomDisplayTitles(), $customDisplay );
+
+		$defaultParams = array(
+			'start_date' => array( '>=' => date( 'Y-m-d' ) ),
```

Comment:
@agh1 Do you still have concerns about down-to-the-second vs same-day calculations as in this context?